app/assets/stylesheets/style.css.scss

```diff
@@ -99,6 +99,12 @@ fieldset.tagline {
 
 #neighborhood_list img {
   border-radius: 10px;
+  opacity: .67;
```

Comment:
`.67` should be written with a leading zero as `0.67`<br>Properties should be ordered border-radius, filter, opacity